app/cdap/text/text-en.yaml

```diff
@@ -3522,6 +3522,20 @@ features:
         failure: Failure
         success: Success
         undo: Undo
-
+    RecipeForm:
+      labels:
+        name: Name of  new recipe
```

Comment:
all the labels should be ordered alphabetically on each level
plz update them and follow the same in all PRs going forward

Reply:
re-ordered labels